### DISCUSSION TITLE
Masque Icons Don't Fill ElvUi Action Bars

### DISCUSSION THREAD

**Discussion by Addvi777** [SFX-WoW/Masque#214] [Issues] [answered]
### Game Flavor

Retail

### Game Version

9.0.1.34492

### Add-On Version

9.0.2

### Describe the Issue

ElvUIesque Masque icons don't fill ElvUi action bars

### Steps To Reproduce

I'm using ElvUi version 12.08 and ElvUIesque Masque icons don't fill ElvUi action bar icons. When you turn off and on Keep Size Ratio in ElvUi panel settings the issue goes away but after relogging it becomes bugged again.

### Screenshots

Bugged icons:
https://imgur.com/a/uESBxVw
Turning off and on Keep Size Ratio in ElvUi panel settings fixes the issue:
https://imgur.com/a/CVVpPDP

**Comment by StormFX** [comment]
It looks like ElvUI is doing something with the scale. Can you try it without ElvUI to see if the issue persists?

**Comment by Addvi777** [comment]
It works fine with other addons like WeakAuras so I guess it's only ElvUi issue.

**Comment by Addvi777** [comment, marked as answer]
Fixed in 12.09 ElvUI update.
